updating integrations destinations local files doccs (#3910)
airbytehq/airbyte: docs/integrations/destinations/local-csv.md

* the local mount is using the `/tmp/airbyte_local` default
* then all data will be written to `/tmp/airbyte_local/cars/models` directory.

> ### Access Replicated Data Files

Reviewer: ```suggestion
## Access Replicated Data Files
```

> * If Airbyte instance is running on the same computer that you are navigating with, you can open your browser to go to [file:///tmp/airbyte\_local](file:///tmp/airbyte_local) and look at the replicated data locally

Reviewer: ```suggestion
If your Airbyte instance is running on the same computer that you are navigating with, you can open your browser and enter [file:///tmp/airbyte\_local](file:///tmp/airbyte_local) to look at the replicated data locally.
```

> * If Airbyte instance is running on a remote server or for some reasons you can't use the first approach ([file:///tmp/airbyte\_local]) to access the replicated files, follow the following steps to access the replicated files:

Reviewer: ```suggestion
If the first approach fails or if your Airbyte instance is running on a remote server, follow the following steps to access the replicated files:
```

>     * Access the scheduler container using `docker exec -it airbyte-scheduler bash`
>     * Navigate to the default local mount using `cd /tmp/airbyte_local`
>     * Navigate to the replicated file directory you specified when you created the destination, using `cd /{Directory_Specified}`
>     * List files containing the replicated data using `ls`
>     * After that you can execute `cat {filename}` to display the data in a particular file

Reviewer: ```suggestion
1. Access the scheduler container using `docker exec -it airbyte-scheduler bash`
2. Navigate to the default local mount using `cd /tmp/airbyte_local`
3. Navigate to the replicated file directory you specified when you created the destination, using `cd /{Directory_Specified}`
4. List files containing the replicated data using `ls`
5. Execute `cat {filename}` to display the data in a particular file
```

Reviewer: @jacqueskpoty just replicate these changes to the local-json file as well!
